src/hotkey/hotkey_item.cpp

```diff
@@ -290,18 +290,21 @@ const std::string hotkey_keyboard::get_name_helper() const
 
 bool hotkey_keyboard::matches_helper(const SDL_Event &event) const
 {
-	if (event.type != SDL_KEYDOWN && event.type != SDL_KEYUP) {
-		return false;
-	}
-
-	SDL_Scancode code;
-	code = event.key.keysym.scancode;
+	unsigned int mods = sdl_get_mods();
 
-	if (code != scancode_) {
+	if (event.type == SDL_TEXTINPUT && text_.length() == 1) {
+		std::string text = std::string(event.text.text);
+		boost::algorithm::to_lower(text);
+		if (text == ":") {
```

Comment:
Because ":" is always going to be internationalisation-dependant no matter how it's approached, but shift is the only key that affects the composition of straight-forward characters.